lib/playground.dart

```diff
@@ -617,7 +617,7 @@ class InlineBracketsColon extends markdown.InlineSyntax {
 }
 
 class InlineBrackets extends markdown.InlineSyntax {
-  InlineBrackets() : super(r'\[\s?((?:.|\n)*?)\s?\](?!\()');
+  InlineBrackets() : super(r'\[\s?((?:.|\n)*?)\s?\](?!\s?\()');
```

Comment:
I know this wasn't originally your code, but a comment explaining what this magic regex is doing would be helpful